integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -352,6 +333,15 @@ object BloopDefaults {
     }.distinct
   }
 
+  def onlyCompilationModules(ms: Seq[Config.Module], classpath: Array[Path]): Seq[Config.Module] = {
+    val classpathFiles = classpath.filter(p => Files.exists(p) && !Files.isDirectory(p))
```

Comment:
I think `Files.isRegularFile` should be at least as good as `Files.exists && !Files.isDirectory` in this case, but I can see the latter being easier to understand (one doesn't need to check the docs to make sure that symbolic links are followed by default).

Reply:
I'll add `Files.isRegularFile` :wink: